modules/4337/hardhat.config.ts

```diff
@@ -66,6 +66,7 @@ const userConfig: HardhatUserConfig = {
   },
   networks: {
     localhost: {
+      url: 'http://localhost:8545',
```

Comment:
For some reason, hardhat refused to connect to the node if this URL wasn't provided. 
According to @nlordell, some IPv4/IPv6 shenanigans could be involved, but I don't think it's worth looking into.

Reply:
VPN maybe? :stuck_out_tongue: